packages/core/src/utils/convert.ts

```diff
@@ -0,0 +1,28 @@
+import type { Address, ChainId, TransactionHash } from '../types'
+
+export function toAddress(address: string): Address {
+  return `0x${address.toLowerCase().replace(/^0x/, '')}` as Address
+}
```

Comment:
What is the reason to strip the 0x and add it back?
Does Typescript require it as `type Address` is `0x${string}`? But then `as Address` should not be required?

Reply:
indeed the `as Address` should not be present, some left over of testing

Reply:
[59eb9d1](https://github.com/liteflow-labs/liteflow-js/pull/142/commits/59eb9d1ebbeb1e439f8b9b949b88409e52644db3)